src/Physics/Plugins/ammoJSPlugin.ts

```diff
@@ -50,10 +53,11 @@ export class AmmoJSPlugin implements IPhysicsEnginePlugin {
 
     /**
      * Initializes the ammoJS plugin
+     * @param _includeSoftBodies when true you can mix tigid and soft bodies (default false)
      * @param _useDeltaForWorldStep if the time between frames should be used when calculating physics steps (Default: true)
      * @param ammoInjection can be used to inject your own ammo reference
      */
-    public constructor(private _useDeltaForWorldStep: boolean = true, ammoInjection: any = Ammo) {
+    public constructor(private _includeSoftBodies: boolean = false, private _useDeltaForWorldStep: boolean = true, ammoInjection: any = Ammo) {

```

Comment:
I was hoping that as Ammo was still in alpha it wouldn't be a problem. The constructors for Oimo and Cannon append from the front 
Oimo  - 'constructor(iterations?: number, oimoInjection = OIMO)'
Cannon - 'constructor(private _useDeltaForWorldStep: boolean = true, iterations: number = 10, cannonInjection = CANNON)'
Ammo - 'constructor(private _includeSoftBodies: boolean = false, private _useDeltaForWorldStep: boolean = true, ammoInjection: any = Ammo)'

Putting it at the end would mean writing in parameters for __useDeltaForWorldStep and ammoInjection as well if the user wanted softbodies!

A possible alternative would be to do a AmmoSoftJSPlugin inheriting from AmmoJSPlugin

Reply:
There are already a good number of samples/playgrounds that might get broken so i would prefer it at the end. Passing undefined allows default parameters to be used so if softbodies are needed some one could do Ammo(undefined, undefined, ... , true). In hindsight it might have been better to have the physics plugins take an object to better support the options as they grow but I don't think it's too bad.

Reply:
Possibly I have been over complicating things (not unusual for me). How about this for an idea - Do away with the _includeSoftBodies flag and set the Ammo physics world to always include soft and rigid bodies. I have tested out a couple of rigid body only examples in a soft and rigid body world and there seems to be no problem.
https://adoring-kirch-5b50fe.netlify.com/rigid1.html
https://adoring-kirch-5b50fe.netlify.com/rigid2.html

@deltakosh and @TrevorDev if you both agree I will submit a new commit with the world set to a soft and rigid one. If you do not agree I will move the _includesoftBodies parameter to the end.

Reply:
Oh that could be great! I let @TrevorDev  weights in

Reply:
Sounds good to me.

Reply:
Will get on with it ☺️

Reply:
Done. Now just one physics world with both rigid and soft bodies